src/admin-views/meta-box.php

```diff
@@ -86,15 +80,35 @@
 			do_action( 'tribe_events_tickets_post_ticket_list', $post_id ); ?>
 
 		</div>
-		<button id="ticket_form_toggle" class="button-secondary ticket_form_toggle"><span class="ticket_form_toggle_text" aria-label="<?php esc_attr_e( 'Add a new ticket' ); ?>"><?php esc_html_e( 'New ticket', 'event-tickets' ); ?></span></button>
-		<button id="rsvp_form_toggle" class="button-secondary ticket_form_toggle"><span class="ticket_form_toggle_text" aria-label="<?php esc_attr_e( 'Add a new RSVP' ); ?>"><?php esc_html_e( 'New RSVP', 'event-tickets' ); ?></span></button>
-		<button id="settings_form_toggle" class="button-secondary"><span class="settings_form_toggle_text"><?php esc_html_e( 'Settings', 'event-tickets' ); ?></span></button>
-	</div>
+		<div>
+			<?php
+			/**
+			 * Allows for the insertion of additional content into the main ticket admin panel after the tickets listing
+			 *
+			 * @param Post ID
```

Comment:
The _ideal_ would be `@param type $var` style - ie `@param int $post_id` (just ensures things are optimal for whenever we start to capture this through WP Parser).